examples/sb3/scenarios/single_agent_tasks/junction_left_turn/scenario.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+from smarts.sstudio.types import Distribution
+from smarts.sstudio import gen_scenario
+from smarts.sstudio import types as t
+
+flow1 = [t.Flow(
+            route=t.Route(
+                begin=("-E3",0, "random"), end=("-E0",0, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        )
+        for _ in range(1)]
+
+flow2 = [t.Flow(
+            route=t.Route(
+                begin=("E0",0, "random"), end=("E3",0, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        )
+        for _ in range(1)]
+
+flow3 = [t.Flow(
+            route=t.Route(
+                begin=("E4", lane, "random"), end=("E1", lane, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        ) for lane in range(2)
+        for _ in range(2)]
+
+flow4 = [t.Flow(
+            route=t.Route(
+                begin=("-E1", lane, "random"), end=("-E4", lane, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        ) for lane in range(2)
+        for _ in range(2)]
+
+flow5 = [t.Flow(
+            route=t.Route(
+                begin=("E0", 0, "random"), end=("E1", 1, "max")
+            ),
+            rate=1,
+            actors={t.TrafficActor("car",speed=Distribution(mean=0.5, sigma=1)): 1},
+            
+        )
+        for _ in range(1)]
```

Comment:
Minor, but we can probably remove the list comprehension part for the ones that have `range(1)` and just make them list literals.

Reply:
oh this `1` in `range(1)` can be easily changed to higher value to control the traffic density.

Reply:
oh this `1` in `range(1)` can be easily changed to higher value to control the traffic density.